cmd/mattermost/commands/channel.go

```diff
@@ -352,10 +347,6 @@ func moveChannelsCmdF(command *cobra.Command, args []string) error {
 	}
 	defer a.Shutdown()
 
-	if len(args) < 2 {
-		return errors.New("Enter the destination team and at least one channel to move.")
-	}
```

Comment:
Should add `Args` to `MoveChannelsCmd`?

Reply:
Looks like I missed it. Will add for this too.

Reply:
Resolved and squashed the commit 👍